### PR TITLE
[BMS] SDK Changed

### LINKED ISSUE #1914: [BMS] SDK Changed
https://github.com/opentelekomcloud/gophertelekomcloud/pull/403

### PR DESCRIPTION
**Pull request by Aloento**
## Summary of the Pull Request

fix #1914

## PR Checklist

* [x] Refers to: #1914
* [x] Tests added/passed.
* [x] Schema updated.
* [x] Release notes added.

## Acceptance Steps Performed

```

```

### CODE REVIEW

**Comment by vladimirvshivkov** [comment]
@Aloento 
basic tests for bms tags fails with following:

```
=== RUN   TestAccBMSTagsV2_basic
    resource_opentelekomcloud_compute_bms_tags_v2_test.go:22: Step 1/1 error: Error running pre-apply refresh: exit status 1
        
        Error: Incorrect attribute value type
        
          on terraform_plugin_test.tf line 22, in resource "opentelekomcloud_compute_instance_v2" "instance_1":
          22:   tags              = ["foo", "bar"]
        
        Inappropriate value for attribute "tags": map of string required.
--- FAIL: TestAccBMSTagsV2_basic (6.95s)

FAIL
```

**Comment by vladimirvshivkov** [comment]
@Aloento 
also, bms server fails:
```
=== RUN   TestAccComputeV2BmsInstance_basic
    resource_opentelekomcloud_compute_bms_server_v2_test.go:22: Step 1/2 error: Error running apply: exit status 1
        
        Error: neither an image ID, or image name were able to be determined
        
          with opentelekomcloud_compute_bms_server_v2.instance_1,
          on terraform_plugin_test.tf line 8, in resource "opentelekomcloud_compute_bms_server_v2" "instance_1":
           8: resource "opentelekomcloud_compute_bms_server_v2" "instance_1" {
        
--- FAIL: TestAccComputeV2BmsInstance_basic (18.29s)

FAIL
```

**Comment by Aloento** [comment]
@vladimirvshivkov thank you.

This problem is not caused by SDK changes. 
I tested with the devel branch and both issues persist.

I will try to figure it out.

File: opentelekomcloud/acceptance/bms/resource_opentelekomcloud_compute_bms_tags_v2_test.go [hunk 1]
```diff
@@ -137,8 +139,8 @@ resource "opentelekomcloud_compute_instance_v2" "instance_1" {
   image_id          = data.opentelekomcloud_images_image_v2.latest_image.id
   security_groups   = ["default"]
   availability_zone = "%s"
-  flavor_id         = "physical.o2.medium"
-  flavor_name       = "physical.o2.medium"
+  flavor_id         = "physical.h2.medium"
+  flavor_name       = "physical.h2.medium"
   tags              = ["foo", "bar"]
```
Comment (anton-sidelnikov) [inline opentelekomcloud/acceptance/bms/resource_opentelekomcloud_compute_bms_tags_v2_test.go:144]: Why not:
```
tags = {
    foo  = "bar"
    john = "doe"
  }
```

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: opentelekomcloud/acceptance/bms/resource_opentelekomcloud_compute_bms_tags_v2_test.go (hunk 1 above)
Comment (Aloento) [reply opentelekomcloud/acceptance/bms/resource_opentelekomcloud_compute_bms_tags_v2_test.go:144]: @anton-sidelnikov 
https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/blob/d5b7cd5487c66237234041e1c7ad38eb3229ac77/opentelekomcloud/services/bms/resource_opentelekomcloud_compute_bms_tags_v2.go#L44-L45

https://github.com/opentelekomcloud/gophertelekomcloud/blob/791d6bc332a31150be5b7f0e7c3ef66c54034feb/openstack/bms/v2/tags/create.go#L10

https://github.com/opentelekomcloud/gophertelekomcloud/blob/0a55ef751ae8be32ab8d07a4bcaa49ac30737da4/openstack/bms/v2/tags/requests.go#L15

File: opentelekomcloud/acceptance/bms/resource_opentelekomcloud_compute_bms_tags_v2_test.go [hunk 2]
```diff
@@ -112,9 +111,12 @@ resource "opentelekomcloud_compute_instance_v2" "instance_1" {
...
   availability_zone = "%s"
-  flavor_id         = "physical.o2.medium"
-  flavor_name       = "physical.o2.medium"
-  tags              = ["foo", "bar"]
+  flavor_id         = "physical.h2.large"
+  flavor_name       = "physical.h2.medium"
+  tags = {
+    foo  = "bar"
+    john = "doe"
+  }
```
Comment (anton-sidelnikov) [inline opentelekomcloud/acceptance/bms/resource_opentelekomcloud_compute_bms_tags_v2_test.go:119]: ok, then why here not tags = ["foo", "bar"]?

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: opentelekomcloud/acceptance/bms/resource_opentelekomcloud_compute_bms_tags_v2_test.go (hunk 2 above)
Comment (Aloento) [reply opentelekomcloud/acceptance/bms/resource_opentelekomcloud_compute_bms_tags_v2_test.go:119]: https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/blob/d5b7cd5487c66237234041e1c7ad38eb3229ac77/opentelekomcloud/services/bms/resource_opentelekomcloud_compute_bms_server_v2.go#L192-L193

**Review by anton-sidelnikov — APPROVED** [review]

**Review by vladimirvshivkov — APPROVED** [review]

**Comment by Aloento** [comment]
How did this automated test pass? 😂😂😂

**Pull request merged**